### PR TITLE
npm-*,pacman-s: add Dutch translation, qm-resize: update Dutch translation

### PR DESCRIPTION
**Pull request by sebastiaanspeck**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page(s) follow the [style guide](/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).

### CODE REVIEW

**Review by leonvsc — APPROVED** [review]

File: pages.nl/common/npm-author.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# npm-author
```
Comment (acuteenvy) [inline pages.nl/common/npm-author.md:1]:
```suggestion
# npm author
```

File: pages.nl/common/npm-owner.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
+# npm-owner
```
Comment (acuteenvy) [inline pages.nl/common/npm-owner.md:1]:
```suggestion
# npm owner
```

File: pages.nl/linux/pacman-s.md [hunk 3]
```diff
@@ -0,0 +1,7 @@
+# pacman -S
+
+> Dit commando is een alias van `pacman --sync`.
+
+- Bekijk de documentatie van het originele commando:
+
+`tldr pacman-sync`
```
Comment (acuteenvy) [inline pages.nl/linux/pacman-s.md:7]:
```suggestion
`tldr pacman sync`
```

**Review by acuteenvy — COMMENTED** [review]
#14824 & #14842

**Review by spageektti — APPROVED** [review]

**Pull request merged**